package.json

```diff
@@ -25,7 +25,7 @@
     "pg": "^7.4.0",
     "pg-hstore": "^2.3.2",
     "request-promise": "^4.2.2",
-    "sequelize": "^4.37.5",
+    "sequelize": "^5.8.12",
```

Comment:
Tedious will need updating as a result of this

Reply:
Hi Floyd, the current sum of dependency-bot directives across Organisations & Directories (etc) point to Sequelize 5.15.1 but npm doco just suggests generic latest tedious (6.6.0) - any insight on where to validate the tedious bump from current 2.6.4 (obv extensive regression testing a good idea...)